app/controllers/users_controller.rb

```diff
@@ -1,4 +1,12 @@
 class UsersController < ApplicationController
+  before_action :logged_in_user, only: [:index, :edit, :update, :destroy]
+  before_action :correct_user, only: [:edit, :update]
+  before_action :admin_user, only: :destroy
+
+  def index
+     @users = Kaminari.paginate_array(User.all).page(params[:page])
+     .per(Settings.show_limit.show_10)  
```

Comment:
Indentation

Reply:
yes

Reply:
No, when you break a line, give add 2 white space before the start of second line in compared with the part of it's first line

Reply:
Can you try with this 
```
User.page(params[:page]).per(Settings.show_limit.show_10)
```

Reply:
@users = Kaminari.paginate_array(User.all).page(params[:page])
      .per(Settings.show_limit.show_10)

Reply:
It exception: undefined method `page' for #<Class:0x007f57b5c253f8>